aprsdigi.c

```diff
@@ -55,7 +56,8 @@ static char copyr[] = "Copyright (c) 1996,1997,1999,2001,2002,2003,2004,2009,201
 #include <time.h>
 #include <signal.h>
 #include <errno.h>
-#include <linux/ax25.h>
+#include <netax25/ax25.h>
```

Comment:
Are these changes consistent across all flavors of linux?

These are not related to the stated purpose of this MR

Reply:
@fweimer-rh  Maybe this should be handled in https://github.com/n2ygk/aprsdigi/blob/master/configure.ac then?

Reply:
I guess I'll go ahead and accept this and we'll see if it breaks anywhere.